plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/shipping-rates-control/index.tsx

```diff
@@ -70,15 +71,39 @@ const ShippingRatesControl = ( {
 	renderOption,
 	context,
 }: ShippingRatesControlProps ): JSX.Element => {
+	const shippingRatesRateCount = getShippingRatesRateCount( shippingRates );
+	const shippingRatesPackageCount =
+		getShippingRatesPackageCount( shippingRates );
+	const previousShippingRatesRateCount = usePrevious(
+		getShippingRatesRateCount( shippingRates )
+	);
+	const previousShippingRatesPackageCount = usePrevious(
+		getShippingRatesPackageCount( shippingRates )
```

Comment:
As above, we already have `shippingRatesPackageCount`. Would it work to reuse it?

Reply:
Fixed in [Reuse variables as parameters](https://github.com/woocommerce/woocommerce/pull/53494/commits/93478679d1e863059b7078f8c255a6523f009eb2) commint.